user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures.md

```diff
@@ -34,6 +34,7 @@ The procedures below can be applied to troubleshoot issues. However, be very car
 - [Managing trend patterns](xref:SLNetClientTest_managing_trend_patterns)
 - [Modifying the engine ID of a DMA](xref:SLNetClientTest_modifying_engine_id)
 - [Removing a DOM manager](xref:SLNetClientTest_removing_DOM_Manager)
+- [Triggering a midnight sync for a DOM manager](xref:SLNetClientTest_triggering_DOM_midnight_sync)
```

Comment:
Almost all items on this page are alphabetically sorted. Should we maintain the alphabetical sorting?